ota_proxy/README.md

```diff
@@ -0,0 +1,81 @@
+# ota-proxy
+# Introduction
+Internal ECUs are not allowed to directly connect to the Internet, so ota-proxy is introduced to allow internal ECUs to apply ota update.
```

Comment:
> Internal ECUs are not allowed to directly connect to the Internet

It depends on the user's environment. so,

`In case of the internal ECUs are not allowed to directly connect to the Internet, ota-proxy is introduced to allow internal ECUs to apply ota update.` should be fine.

Reply:
@Bodong-Yang 
Can you fix this sentence?

Reply:
OK!